protocol-designer/src/steplist/formLevel/stepFormToArgs/flexStackerFormToArgs.ts

```diff
@@ -25,6 +25,7 @@ export const flexStackerFormToArgs = (
         commandCreatorFnName: 'flexStackerFillItems',
         fillLabwareUri: fillLabwareUri,
         fillQuantity: fillQuantity,
+        interventionMessage: interventionMessage,
```

Comment:
can go shorthand with these right
```suggestion
fillLabwareUri,
fillQuantity,
interventionMessage,
```